### PR TITLE
Fixed error thrown when updating existing flexible data

### PR DESCRIPTION
**Pull request by devangelito**
Fixes this issue - https://github.com/vudodov/json/issues/12

### CODE REVIEW

File: V.Udodov.Json/Entity.cs [hunk 1]
```diff
@@ -96,15 +96,15 @@ private void Set(string key, object value)
             if (_extensionDataJsonSchema != null)
             {
                 var obj = JObject.FromObject(_data);
-                obj.Add(key, token);
+                obj[key] = token;
```
Comment (vudodov) [inline V.Udodov.Json/Entity.cs:99]: 🔥

File: V.Udodov.Json.Tests/EntityTests.cs [hunk 2]
```diff
@@ -55,6 +55,29 @@ public void WhenSettingEntityFlexibleDataWithSchemaAndDataIsInvalidItShouldThrow
                 .Should().HaveCount(1);
         }
 
+        [Fact]
+        public void WhenUpdatingFlexibleDataItShouldUpdateWithoutErrors()
```
Comment (vudodov) [inline V.Udodov.Json.Tests/EntityTests.cs:59]: Nice test, man

**Review by vudodov — APPROVED** [review]
@devangelito awesome fix and great test!

**Comment by devangelito** [comment]
My pleasure, man. Thanks for accepting my changes. :)

**Pull request merged**